cob_teleop/ros/src/cob_teleop.cpp

```diff
@@ -369,6 +382,20 @@ void CobTeleop::updateBase()
   }
 }
 
+void CobTeleop::updateArm()
```

Comment:
why is the function not called e.g. `freedriveArm`?

Reply:
i thought maybe there will be more function added to the cob teleop connected to the arm. Than the main has two functions `updateBase` and `updateArm`